nodejs/week4/spam-detector/index.js

```diff
@@ -0,0 +1,70 @@
+class Email {
+  constructor(subject, body) {
+    this.subject = subject;
+    this.body = body;
+  }
+}
+
+const spamWords = ["viagra", "offer", "free", "business proposal"];
+
+class SpamDetector {
+  isSpam(email) {
+    const subject = email.subject;
+    const body = email.body;
+    const string = subject + body;

```

Comment:
`string` doesn't represent the content of the `const` really well. Maybe you could call it `contentToCheck`?

Reply:
changed